tests/scripts/icmp_responder.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+
+from scapy.all import conf, Ether, ICMP, IP
+from scapy.arch import get_if_hwaddr
+from scapy.data import ETH_P_IP
+from select import select
+
+
+def respond_to_icmp_request(socket, request):
+    """Respond to ICMP request."""
+    reply = request.copy()
+    reply[ICMP].type = 0
```

Comment:
You may need to subtract `8` from the ICMP checksum.

Reply:
Sure, but could you please explain why we need to subtract 8?

Reply:
Mark as resolved, `scapy` will take care of the recalculation when setting `chksum` to `None`